README.md

```diff
@@ -110,6 +110,14 @@ Get the `activeSpan` in the `Context`.
 go2sky.ActiveSpan(ctx)
 ```
 
+## With Span
```

Comment:
Could you explain a little more about the use case? Is this span from some other context?

Reply:
Like this, need to save the `span` to multiple context.

https://github.com/arugal/functions-framework-go/blob/patch/plugin/skywalking/sync-request.go#L21-L22